modules/migration-dvm-error-node-selectors.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+// * migration/migrating_3_4/troubleshooting-3-4.adoc
+// * migration/migrating_4_1_4/troubleshooting-4-1-4.adoc
+// * migration/migrating_4_2_4/troubleshooting-4-2-4.adoc
+
+[id="migration-dvm-error-node-selectors_{context}"]
+= Direct volume migration does not complete
+
+If direct volume migration does not complete, the target cluster might not have the same `node-selector` annotations as the source cluster.
+
+{mtc-short} migrates namespaces with all annotations in order to preserve security context constraints and scheduling requirements. During direct volume migration, {mtc-short} creates Rsync transfer pods on the target cluster in the namespaces that were migrated from the source cluster. If a target cluster namespace does not have the same annotations as the source cluster namespace, the Rsync transfer pods cannot be scheduled. They remain in a `Pending` state.
+
+You can identify and fix this issue by performing the following procedure.
+
+.Procedure
+
+. Check the status of the `MigMigration` pod:
++
+[source,terminal]
+----
+$ oc describe migmigration <88435fe0-c9f8-11e9-85e6-5d593ce65e10> -n openshift-migration <1>
```

Comment:
Could the `<88435fe0-c9f8-11e9-85e6-5d593ce65e10>` replaceable be more of a description than an example itself? E.g., `oc describe migmigration <pod_name>`?